src/graphql/typeDefs/dataTypes/partner.graphql

```diff
@@ -1,10 +1,9 @@
 type Partner @key(fields: "id") {
   id: ID!
 
-  year: Int
+  year: Int @deprecated(reason: "belongs on the event service")
```

Comment:
Is there any benefit to marking this deprecated vs just removing it?

Reply:
I thought it was being used in the website but maybe not. I was going to come back to it after we updated the site.